locale/zh-cn/site.json

```
@@ -1,5 +1,6 @@
 {
   "title": "Node.js",
+  "editOnGithub":"在 GitHub 中直接编辑",
```

```suggestion
  "editOnGithub":"在 GitHub 上编辑",
```

I would prefer to translate literally.